go.mod

```diff
@@ -41,18 +42,16 @@ require (
 	github.com/mitchellh/mapstructure v1.1.2 // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/omnition/scribe-go v0.0.0-20190131012523-9e3c68f31124
-	github.com/onsi/ginkgo v1.7.0 // indirect
-	github.com/onsi/gomega v1.4.3 // indirect
 	github.com/opentracing/opentracing-go v1.0.2 // indirect
-	github.com/openzipkin/zipkin-go v0.1.3
+	github.com/openzipkin/zipkin-go v0.1.6
 	github.com/orijtech/prometheus-go-metrics-exporter v0.0.3-0.20190313163149-b321c5297f60
 	github.com/orijtech/promreceiver v0.0.5
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/philhofer/fwd v1.0.0 // indirect
 	github.com/pkg/errors v0.8.0
 	github.com/prashantv/protectmem v0.0.0-20171002184600-e20412882b3a // indirect
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/procfs v0.0.0-20181204211112-1dc9a6cbc91a
+	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
```

Comment:
I'd stay away from Prometheus version updates. In the past they once made all floats into ints, then reverted, now it seems like they've unreverted and you can see a failure here https://travis-ci.org/census-instrumentation/opencensus-service/builds/515417341#L1241

Reply:
Unfortunately this is a consequence of running `GO111MODULE=on go mod tidy`. I think I have to fix the test failure again.